elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchEdge.java

```diff
@@ -0,0 +1,83 @@
+package org.vertexium.elasticsearch5;
+
+import org.vertexium.*;
+import org.vertexium.mutation.ExistingEdgeMutation;
+
+public class ElasticsearchEdge extends ElasticsearchElement implements Edge {
+    private String className = ElasticsearchEdge.class.getSimpleName();
```

Comment:
`static final`, all caps, use `.getName()`

Reply:
probably don't actually need the class name on exceptions since it will be in the stack trace.